easybuild/easyconfigs/u/udocker/udocker-1.3.4-fosscuda-2020b-Python-3.8.6.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonPackage'
```

Comment:
This file can be removed alltogether, see the instructions at https://docs.easybuild.io/en/latest/Integration_with_GitHub.html#deleting-easyconfig-files-or-patches on how to delete easyconfig files if you use EasyBuild's github integration (if you don't use the github integration, you can simply `git rm` this file from your local fork, push that change to your local fork, and this PR should update itself automatically).